xla/hlo/ir/hlo_computation.h

```diff
@@ -26,6 +26,8 @@ limitations under the License.
 #include "absl/container/flat_hash_map.h"
 #include "absl/container/flat_hash_set.h"
 #include "absl/functional/function_ref.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
```

Comment:
I think this header needs corresponding target to be added in the BUILD file

Reply:
Done